robottelo/config/facade.py

```diff
@@ -284,6 +347,21 @@ def _dispatch_computed_value(self, key):
         self._add_to_cache(key, value)
         return value
 
+    def _dispatch_robottelo_value(self, key):
+        robottelo_keys = [setting.lower() for setting in self._configs[0].robottelo.keys()]
+        top_key = key.split('.')[0] if '.' in key else key
```

Comment:
```suggestion
        top_key = key.split('.')[0]
```

Reply:
Its required when key == `satttools.rhel6` but not when key == `capsule_repo` which doesnt have `.`.

In short, it is required for 4 level of a setting. E.g `settings.robottelo.sattools.rhel6`.

Reply:
@JacobCallahan you can check Robottelo.yaml for those kind of settings.

Reply:
From a pure python perspective, this change isn't necessary. If no period is in they key, then split will result in a single element list. Since you just want that first element, it is functionally the same.

Reply:
Yes ! Updated.